docs/conf.py

```diff
@@ -24,251 +17,32 @@
 
 os.environ["DJANGO_SETTINGS_MODULE"] = "tests.settings"
 
-import django
-import mozilla_django_oidc
+import django  # noqa
 
+import mozilla_django_oidc  # noqa
```

Comment:
These won't work unless `docs/requirements.txt` include all the dev requirements (e.g. django). If I create a fresh virtual environment, install `docs/requirements.txt`, and then do `make docs`, I get this:

```
$ make docs
rm -rf docs/source
sphinx-apidoc -o docs/source/ mozilla_django_oidc
make -C docs clean
make[1]: Entering directory '/home/willkg/mozilla/mozilla-django-oidc/docs'
rm -rf _build/*
make[1]: Leaving directory '/home/willkg/mozilla/mozilla-django-oidc/docs'
make -C docs html
make[1]: Entering directory '/home/willkg/mozilla/mozilla-django-oidc/docs'
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v7.2.6

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/willkg/.pyenv/versions/3.10.12/lib/python3.10/site-packages/sphinx/config.py", line 358, in eval_config_file
    exec(code, namespace)  # NoQA: S102
  File "/home/willkg/mozilla/mozilla-django-oidc/docs/conf.py", line 20, in <module>
    import django  # noqa
ModuleNotFoundError: No module named 'django'

make[1]: *** [Makefile:53: html] Error 2
make[1]: Leaving directory '/home/willkg/mozilla/mozilla-django-oidc/docs'
make: *** [Makefile:38: docs] Error 2
```

Reply:
Apologies for the back n forth here. I neglected to destroy my virtual environment which already had the packages. With the update `make docs` build without any errors for me. Hopefully one more pass :) ?